endpoints/setuid_test.go

```diff
@@ -288,6 +288,270 @@ func TestSetUIDEndpoint(t *testing.T) {
 	}
 }
 
+func TestExtractGDPRInfo(t *testing.T) {
+	type testOutput struct {
+		requestInfo gdpr.RequestInfo
+		err         error
+	}
+	type testCase struct {
+		desc     string
+		inUri    string
+		expected testOutput
+	}
+	testSuite := []struct {
+		sDesc string
```

Comment:
Is this `sDesc` value used anywhere in these tests? If not, could it be removed?

Reply:
Corrected in [df562e9](https://github.com/prebid/prebid-server/pull/2761/commits/df562e921419d2863a63430fde9c290a49eec434)